.github/workflows/build-docs.yml

```diff
@@ -42,11 +42,17 @@ jobs:
           mamba update -n stardis --file conda-linux-64.lock
         if: steps.cache-env-lockfile.outputs.cache-hit != 'true'
         
+      - name: Install TARDIS
+        id: install-tardis
+        # shell: bash -l {0}
+        run: |
+          pip install git+https://github.com/tardis-sn/tardis.git@release-2023.04.16
```

Comment:
Yes. This line can be changed easily to use the most recent TARDIS release. 

I recall @smithis7 had some justification for this, but I don't remember what it was

Reply:
Does the conda lock file specify a tardis version? They should be synchronized so that tests pass on the environment that stardis is installed into. Or just use the lock file to install the environment for tests/docs the same way tardis itself does.

Reply:
The short answer is no, although the answer is actually more subtle. 

The tardis version is specified in [stardis_env3.yml](https://github.com/tardis-sn/stardis/blob/4b24b60ac91b6562b6b771fddca8983409b02d77/stardis_env3.yml#L44), but [gets lost when creating platform-specific lockfiles](https://conda.github.io/conda-lock/output/#explicit-lockfile). I believe this will be implemented in the future, at which point, we can just rerun [these commands](https://smokestacklightnin.github.io/stardis/branch/virtual-environment/make-conda-lock-reproducible/contributing.html#generating-conda-lockfiles) to include tardis in the platform-specific lockfile. 

Does that answer your question

Reply:
It's also worth nothing that invoking [the `setup.py` interface is legacy](https://pip.pypa.io/en/stable/reference/build-system/setup-py/)

[Here is a longer explanation called "Why you shouldn't invoke setup.py directly"](https://pip.pypa.io/en/stable/reference/build-system/setup-py/)

Reply:
Alright, makes sense to me. Will leave this unresolved so others can read it easily and weigh in.